core/src/test/java/org/folio/tlib/util/TenantUtilTest.java

```diff
@@ -40,9 +40,13 @@ public void invalidNull() {
 
   @Test
   public void invalid() {
-    List.of("", "1", "1abc", "abcdefghijklmnopqrstuvwxyz789012").forEach(tenant -> {
+    List.of("", "1", "1abc", "_a", "a_b", "du_").forEach(tenant -> {
       Throwable t = assertThrows(IllegalArgumentException.class, () -> tenant(tenant));
       assertThat(t.getMessage(), containsString(" must match "));
     });
+    List.of("abcdefghijklmnopqrstuvwxyz789012").forEach(tenant -> {
```

Comment:
Why is a list and for each needed for a single example?

Reply:
Probably because there were multiple before. A big deal?

Reply:
> Probably because there were multiple before. 

According to the diff this whole section is brand new code.

> A big deal?

That isn't up to me. You asked for my feedback, it is up to you as to whether you agree with it or want to do anything with it.

Reply:
Changed.